functions.php

```diff
@@ -511,6 +562,7 @@ function get_sql_add_post($type_content) : string {
 
 /**
  * Возвращает id поста
+ * @param msqli
```

Comment:
```suggestion
 * @param msqli $link
```

Тут и ниже пропустил имя аргумента